driver/bpf/fillers.h

```diff
@@ -2818,8 +2818,33 @@ FILLER(execve_extra_tail_2, true) {
 	} else {
 		res = bpf_push_empty_param(data);
 	}
+	CHECK_RES(res);
 
-	return res;
+	/* Parameter 29: pgid (type: PT_UID) */
+	pid_t pgid = 0;
+	// this is like calling in the kernel:
+	//
+	// struct pid *grp = task_pgrp(current);
+	// int pgrp = pid_nr(grp);
+#ifdef HAS_TASK_PIDS_FIELD
```

Comment:
Extracting a small helper function here since same code is also used below in `sched_prog_exec_5`?

Reply:
done ty!